clients/command/src/main/java/org/eclipse/hono/client/command/CommandContext.java

```diff
@@ -117,11 +118,14 @@ default void reject(final String error) {
      *
      * @param tracer The tracer to use.
      * @param command The command for which the span should be started.
-     * @param spanContext Existing span context.
+     * @param spanContext The span context to set as parent span.
```

Comment:
how about renaming this to *parentSpan* then ...

Reply:
Renamed to `parentSpanContext` - I'd rather make it clear that it is a span _context_, not a span here (as used in some other places as method parameter).